src/index.html

```diff
@@ -3,11 +3,359 @@
+          <li><a href="#hire-us" class="nav__link">Contact us</a></li>
+        </ul>
+      </nav>
+      <a href="#hire-us" class="nav__hire-us">Hire us</a>
+    </header>
+
+    <aside class="menu" id="menu">
+      <div class="header__icons">
+        <a href="#">
+          <img src="images/logo/Air.png" alt="logo">
+        </a>
+        <a href="#" class="icon icon--close"></a>
+      </div>
+      </div>
+      <nav class="menu__nav">
+        <ul class="menu__nav--list">
+          <li><a href="#about-us" class="nav__link">About us</a></li>
+          <li><a href="#" class="nav__link">home</a></li>
+          <li><a href="#who-we-are" class="nav__link">Work</a></li>
+          <li><a href="#services" class="nav__link">Services</a></li>
+          <li><a href="#hire-us" class="nav__link">Contact us</a></li>
+        </ul>
+      </nav>
+    </aside>
+
+    <section class="strategic-agency__block" id="about-us">
+      <div class="strategic-agency--top">
+        <h1>Strategic Agency</h1>
+        <span>We believe in the power of bold ideas that can solve business challenges.</span>
+        <button class="btn" href="#">Learn more</button>
+      </div>
+
+      <div class="strategic-agency--bottom">
+        <div class="carousel">
+          <div class="carousel__track-container">
+            <ul class="carousel__track">
+              <li class="carousel__slide current-slide">
+                <img
+                src="images/slider/image.png"
+                alt="slider-image"
+                class="carousel__slide__photo">
+              </li>
+              <li class="carousel__slide">
+                <img
+                src="images/slider/image2.png"
+                alt="slider-image"
+                class="carousel__slide__photo">
+              </li>
+              <li class="carousel__slide">
+                <img
+                src="images/slider/image3.png"
+                alt="slider-image"
+                class="carousel__slide__photo">
+              </li>
+            </ul>
+          </div>
+          <button class="carousel__button carousel__button--left" href="#">
+            <img src="images/slider/arrow-left.svg" alt="arrow-left">
+          </button>
+          <button class="carousel__button carousel__button--right" href="#">
+            <img src="images/slider/arrow-right.svg" alt="arrow-right">
+          </button>
+          <div class="carousel__button__wrapper">
+            <p>intro</p>
+            <span>By the same illusion which lifts the<br>horizon.</span>
+          </div>
+        </div>
+      </div>
+    </section>
+
+    <section class="who-we-are" id="who-we-are">
+      <h3>Who we are</h3>
+      <p>We embrace a strategic approach to creative ideas. We are interested in people and human relationships. This is the main thing you need to know about us. We believe in the power of bold ideas that can solve business challenges.</p>
```

Comment:
This element should have a set width allowing its text to be clearly read in any screen size, check it out:

Yours:
![image](https://github.com/mate-academy/layout_dia/assets/52002856/a019ae9b-b3af-4021-ba02-0b1a3ac4f9be)

Figma:
![image](https://github.com/mate-academy/layout_dia/assets/52002856/4316a14c-1f62-40bd-a955-a54e81091e34)